src/bundle/Resources/config/services.yaml

```
@@ -17,5 +17,6 @@ imports:
     - { resource: services/mappers.yaml }
     - { resource: services/response.yaml }
     - { resource: services/services.yaml }
+    - { resource: services/strategies.yaml }
```

Missing file import (I assume you want it added, since it's part of PR)

```suggestion
    - { resource: services/storages.yaml }
    - { resource: services/strategies.yaml }
```